contracts/core/modules/RebalanceAuctionModule.sol

```diff
@@ -197,4 +226,98 @@ contract RebalanceAuctionModule is
             componentTransferAmount
         );
     }
+
+    /* ============ Public Functions ============ */
+
+    /**
+     * Place bid on Rebalancing Set Token and return token flows. 
+     *
+     * @param  _rebalancingSetToken    Address of the rebalancing token being bid on
+     * @param  _quantity               Number of currentSets to rebalance
+     * @return combinedTokenArray      Array of token addresses invovled in rebalancing
+     * @return inflowUnitArray         Array of amount of tokens inserted into system in bid
+     * @return outflowUnitArray        Array of amount of tokens taken out of system in bid
+     */
+    function placeBidAndGetTokenFlows(
+        address _rebalancingSetToken,
+        uint256 _quantity
+    )
+        private
+        returns (address[], uint256[], uint256[])
+    {
+        // Make sure the rebalancingSetToken is tracked by Core
+        require(
+            coreInstance.validSets(_rebalancingSetToken),
+            "RebalanceAuctionModule.bid: Invalid or disabled SetToken address"
+        );
+
+        // Receive addresses of tokens involved and arrays of inflow/outflow associated with each token
+        return IRebalancingSetToken(_rebalancingSetToken).placeBid(_quantity);
+    }
+
+    /**
+     * Settle bid token flows by attributing tokens to bidder in Vault. 
+     *
+     * @param  _rebalancingSetToken    Address of the rebalancing token being bid on
+     * @param  _tokenArray             Array of token addresses invovled in rebalancing
+     * @param  _inflowUnitArray        Array of amount of tokens inserted into system in bid
+     * @param  _outflowUnitArray       Array of amount of tokens taken out of system in bid
+     */
+    function settleToVault(
+        address _rebalancingSetToken,
+        address[] memory _tokenArray,
+        uint256[] memory _inflowUnitArray,
+        uint256[] memory _outflowUnitArray
+    )
+        private
+    {
+        // Retrieve tokens from bidder and deposit in vault for rebalancing set token
+        coreInstance.batchDepositModule(
```

Comment:
What's the tradeoffs between in-lining this logic vs. making it its own function?